operator/cmd/mesh/profile-common.go

```diff
@@ -63,7 +63,7 @@ func GenerateConfig(inFilenames []string, setOverlayYAML string, force bool, kub
 	}
 
 	errs, warning := validation.ValidateConfig(false, iops.Values, iops)
-	if warning != "" {
+	if !force && warning != "" {
```

Comment:
I think we should still see warnings even if force is enabled

Reply:
otherwise, we may need to add one parameter for this function to prevent warnings on generating the old (installed) version configs.

Reply:
isn't that desired? If my old settings now produce warnings seems like I should know about it?

Reply:
not in this scenario. Here during upgrade, we get the current installed version 1.5.1.
Then we use the binary version (1.6.0) code to validate IOP for installed version (1.5.1) default configs by downloading the 1.5.1's packages. Those defaults are built-in the 1.5.1 package, not from the user input.